templates/test/ci/cluster-template-prow-machine-pool-flex.yaml

```diff
@@ -101,11 +101,13 @@ spec:
       nodeRegistration:
         kubeletExtraArgs:
           cloud-provider: external
+          youwillfailtobootstrappleaseandthankyou: ""
```

Comment:
This PR is [WIP] while I make sure that VMSS instances that fail to bootstrap really do get logs now.

Reply:
logs!
```
Jun 18 05:38:36.422585 capz-e2e-d2pftj-vmss-mp-0000000 kubelet[1551]: E0618 05:38:36.422433    1551 run.go:72] "command failed" err="failed to parse kubelet flag: unknown flag: --youwillfailtobootstrappleaseandthankyou"
```
and on windows:
```
E0618 06:13:42.140420    2456 run.go:72] "command failed" err="failed to parse kubelet flag: unknown flag: --youwillfailtobootstrappleaseandthankyou"
```

Reverted these template changes.

/retitle Make e2e node log collection more self-sufficient